src/mempool_args.cpp

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2022 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <mempool_args.h>
+
+#include <mempool_options.h>
+#include <util/system.h>
+
+namespace {
+void ApplyArgsManOptions(const ArgsManager& argsman, MemPoolLimits& mempool_limits)
+{
+    mempool_limits.ancestor_count = argsman.GetIntArg("-limitancestorcount", mempool_limits.ancestor_count);
```

Comment:
nit in 6843403a15cab34bc4e1e7acf7a0e509a2d437df:

```suggestion
    if (auto cnt = argsman.GetIntArg("-limitancestorcount") mempool_limits.ancestor_count = *cnt;
```

Any reason to use a different style for those, when it is less code to keep the style consistent with the other parsing?

Reply:
I wanted to make visually clear the cases where we directly use the `ArgsMan` arg in `CTxMemPool::Options` and which cases get transformed before getting set in `CTxMemPool::Options`. However, I don't feel strongly so let me know if you prefer the consistency instead!

Reply:
I don't care too much. My suggestion uses less code, but this is only a style nit.